css-transforms-1/Overview.bs

```diff
@@ -1092,45 +1092,69 @@ The scaling causes a non-invertible CTM for the coordinate space of the div box.
+        then the value ''scale(1)'' will be used for <var>V<sub>b</sub></var>
+        and interpolation will proceed using the next rule.
+        Similarly, if <var>V<sub>a</sub></var> is ''scale(1)''
+        and <var>V<sub>b</sub></var> is ''scale(2) rotate(50deg)''
+        then the interpolation will be performed as if
+        <var>V<sub>a</sub></var> were ''scale(1) rotate(0)''.
     </div>
 
-  <li id="transform-transform-animation">
-    If <em>from-</em> and <em>to-transform</em> have the same number of transform functions, each transform function pair has either the same name, or is a derivative of the same <a href="#transform-primitives">primitive</a>:
-		* Interpolate each transform function pair as described in <a href="#interpolation-of-transform-functions">Interpolation of transform functions</a>. The computed value is the resulting transform function list.
-
-    <div class="example">
-        For example, if <em>from-transform</em> is ''scale(1) translate(0)'' and <em>to-transform</em> is ''translate(100px) scale(2)'' then ''scale(1)'' and ''translate(100px)'' as well as ''translate(0)'' and ''scale(2)'' don't share a common primitive and therefore can not get interpolated following this rule.
-
+  <li>
+    Let <var>V<sub>result</sub></var> be an empty list.
+    Beginning at the start of
+    <var>V<sub>a</sub></var> and <var>V<sub>b</sub></var>,
+    compare the corresponding functions at each position:
+      * While the functions have either the same name,
+        or are derivatives of the same
+        [[#transform-primitives|primitive transform function]],
+        interpolate the corresponding pair of functions as described in
+        [[#interpolation-of-transform-functions]]
+        and append the result to <var>V<sub>result</sub></var>.
+      * If the pair do not have a common name
+        or [[#transform-primitives|primitive transform function]],
+        post-multiply the remaining transform functions in each of
+        <var>V<sub>a</sub></var> and <var>V<sub>b</sub></var> respectively
+        to produce two 4x4 matrices.
+        [=Interpolate=] these two matrices as described in
+        [[#matrix-interpolation]],
+        append the result to <var>V<sub>result</sub></var>,
+        and cease iterating over
+        <var>V<sub>a</sub></var> and <var>V<sub>b</sub></var>.
+
+    <div class=example>
+        For example,
+        if <var>V<sub>a</sub></var> is ''scale(1) rotate(90deg) translate(20px)''
+        and <var>V<sub>b</sub></var> is ''scale(2) translate(10px) rotate(270deg)'',
+        the ''scale(1)'' and ''scale(2)'' functions will be interpolated
+        according to [[#interpolation-of-transform-functions]]
+        while the remainder of each list--
+        ''rotate(90deg) translate(20px)'' and ''translate(10px) rotate(270deg)''--
+        will first be converted to equivalent 4x4 matrices
+        and then interpolated as described in [[#matrix-interpolation]].
```

Comment:
This example, if I'm thinking correctly, gives the same result whether you use whole-property matrix interpolation or this smarter interpolation.  I recommend providing an example that showcases the differences between the two; an obvious example would be having the common prefix be a `rotate(0deg)` and `rotate(360deg)`, which is hugely different.

Then you can also have a small note in this that the spec previously used whole-property matrix interpolation, which would have a different effect.